### PR TITLE
add test & solution

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

**Review by zainabdnaya — APPROVED** [review]
lgtm

**Review by MSilva95 — CHANGES_REQUESTED** [review]
Fix this commits and then i will test it

File: tests/reversestrings_test/main.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package main
```
Comment (MSilva95) [inline tests/reversestrings_test/main.go:1]: format the code and its good to go...

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: tests/reversestrings_test/main.go (hunk 1 above)
Comment (Hamzaelkhatri) [reply tests/reversestrings_test/main.go:1]: fixed

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**